setup.py

```diff
@@ -1,12 +1,10 @@
 #!/usr/bin/env python
 # -*- coding: utf-8 -*-
 
-import versioneer
+import setuptools
 
-try:
-    from setuptools import setup
-except ImportError:
-    from distutils.core import setup
```

Comment:
No point in trying to run without setuptools nowadays, especially for scikit-build's own wheel build.